crates/forge/Cargo.toml

```diff
@@ -45,15 +43,15 @@ rand.workspace = true
 scarb-metadata.workspace = true
 scarb-ui.workspace = true
 semver.workspace = true
-schemars = { version = "0.8.12", features = ["preserve_order"] }
+schemars.workspace = true
 cairo-vm.workspace = true
-ark-ff = "0.4.0-alpha.7"
-ark-std = "0.4.0"
-ark-secp256k1 = "0.4.0"
-ark-secp256r1 = "0.4.0"
+ark-ff.workspace = true
+ark-std.workspace = true
+ark-secp256k1.workspace = true
+ark-secp256r1.workspace = true
 # openssl is being used, please do not remove it!
```

Comment:
This comment can be moved to the top Cargo.toml

Reply:
I don't think so (I did so at first) I think it should be here because it is used invisibly by **this** package. In workspace we should only keep comments regarding the versions etc.